### DISCUSSION TITLE
jpa가 뭔가요?

### DISCUSSION THREAD

**Discussion by tjdud22** [starshb/pharmacy_Find#2] [Q&A] [answered]
jpa가 뭔가요?

**Comment by starshb** [comment, marked as answer]
JPA(Java Persistence API)
- Java 진영에서 ORM(Object-Relational Mapping) 기술 표준으로 사용하는 인터페이스 모음
- 자바 어플리케이션에서 관계형 데이터베이스를 사용하는 방식을 정의한 인터페이스
- 인터페이스 이기 때문에 Hibernate, OpenJPA 등이 JPA를 구현함
